tests/system_tests/conftest.py

```diff
@@ -493,115 +415,26 @@ def wandb_verbose(request):
     return request.config.getoption("--wandb-verbose", default=False)
 
 
-@dataclasses.dataclass
-class UserFixtureCommand:
-    command: Literal["up", "down", "down_all", "logout", "login", "password"]
-    username: Optional[str] = None
-    password: Optional[str] = None
-    admin: bool = False
-    method: Literal["post"] = "post"
-
-    def address(self, addr: LocalWandbBackendAddress) -> str:
-        return urllib.parse.urljoin(addr.fixture_service_url, "db/user")
-
-
-def random_string(length: int = 12) -> str:
-    """Generate a random string of a given length.
-
-    :param length: Length of the string to generate.
-    :return: Random string.
-    """
-    return "".join(
-        secrets.choice(string.ascii_lowercase + string.digits) for _ in range(length)
-    )
-
-
-@pytest.fixture(scope="session")
-def user_factory(worker_id: str):
-    def _user_factory(fixture_fn):
-        username = f"user-{worker_id}-{random_string()}"
-        command = UserFixtureCommand(command="up", username=username)
-        fixture_fn(command)
-        command = UserFixtureCommand(
-            command="password",
-            username=username,
-            password=username,
-        )
-        fixture_fn(command)
-
-        with unittest.mock.patch.dict(
-            os.environ,
-            {
-                "WANDB_API_KEY": username,
-                "WANDB_ENTITY": username,
-                "WANDB_USERNAME": username,
-            },
-        ):
-            yield username
-
-            command = UserFixtureCommand(
-                command="down",
-                username=username,
-            )
-            fixture_fn(command)
-
-    yield _user_factory
-
-
-@pytest.fixture(scope="session")
-def fixture_fn_factory():
-    def _fixture_fn_factory(address: LocalWandbBackendAddress):
-        def fixture_util(cmd: UserFixtureCommand) -> bool:
```

Comment:
**note:** Most of this was absorbed into `BackendFixtureFactory` in `backend_fixtures.py`.  This seemed like it could be easier to maintain & introspect if flattened, but open to pushback and/or understanding if I missed anything